ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.test.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import configureMockStore from 'redux-mock-store';
+import sinon from 'sinon';
+import { renderWithProvider } from '../../../../../test/lib/render-helpers';
+import EditGasToolTip from './edit-gas-tooltip';
+
+const LOW_GAS_OPTION = {
+  gasLimit: 21000,
+  maxFeePerGas: '2.010203381',
+  maxPriorityFeePerGas: '1.20004164',
+  maxFeePerGasValue: '2.383812808',
+  maxPriorityFeePerGasValue: '1.5',
+  origin: 'metamask',
+};
+
+const MEDIUM_GAS_OPTION = {
+  gasLimit: 21000,
+  maxFeePerGas: '2.383812808',
+  maxPriorityFeePerGas: '1.5',
+  maxFeePerGasValue: '2.383812808',
+  maxPriorityFeePerGasValue: '1.5',
+  origin: 'metamask',
+};
+
+const HIGH_GAS_OPTION = {
+  gasLimit: 21000,
+  maxFeePerGas: '2.920638342',
+  maxPriorityFeePerGas: '2',
+  maxFeePerGasValue: '2.383812808',
+  maxPriorityFeePerGasValue: '1.5',
+  origin: 'metamask',
+};
+
+const renderComponent = (props) => {
+  const mockStore = {
+    metamask: {
+      provider: {
+        type: 'test',
+      },
+    },
+  };
+
+  const store = configureMockStore()(mockStore);
```

Comment:
May be use `configureStore` like here: https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.test.js#L50

@mcmire suggested this for unit tests and I think is better over using mock store.